salt/states/pkgrepo.py

```diff
@@ -212,6 +213,9 @@ def managed(name, ppa=None, copr=None, aptkey=True, **kwargs):
     mirrorlist
         A URL which points to a file containing a collection of baseurls
 
+    metalink
+        A URL for a curated list of non-stale mirrors
```

Comment:
can you add .. versionadded:: 3006 for this new argument.

Reply:
Added, I think.